.github/workflows/main.yaml

```diff
@@ -38,11 +38,13 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0 # all
-      - name: Setup .NET 6.0
+
```

Comment:
Because the tests target both 6 and 8, we should have both a setup net6 and a setup net8 action here